agents-manage-ui/src/components/sidebar-nav/project-switcher.tsx

```diff
@@ -1,56 +1,118 @@
+  icon: FC<ComponentProps<'svg'>> | false;
+}> = ({ name, description, icon: Icon }) => {
+  return (
+    <>
+      <Avatar className="h-8 w-8 rounded-lg">
+        <AvatarFallback className="rounded-lg uppercase">{name.slice(0, 2)}</AvatarFallback>
+      </Avatar>
+      <div className="grid flex-1 text-left text-sm leading-tight">
+        <span className="truncate font-medium">{name}</span>
+        <span className="truncate text-xs">{description}</span>
+      </div>
+      {Icon && <Icon className="ml-auto size-4" />}
+    </>
+  );
+};
 
 export function ProjectSwitcher() {
   const [isLoading, setIsLoading] = useState(true);
   const [projects, setProjects] = useState<Project[]>([]);
-  const params = useParams();
-  const tenantId = params.tenantId;
-  const projectId = params.projectId;
+  const [isProjectDialogOpen, setIsProjectDialogOpen] = useState(false);
+  const { tenantId, projectId } = useParams<{ tenantId: string; projectId: string }>();
+  const { isMobile } = useSidebar();
+
+  const handleCreateProject = useCallback(() => {
+    setIsProjectDialogOpen(true);
+  }, []);
 
   useEffect(() => {
     if (!tenantId) return;
 
-    fetchProjectsAction(tenantId as string)
-      .then((res) => {
-        setIsLoading(false);
-        if (res.success && res.data) {
-          const projectResults = res.data;
-          setProjects(projectResults);
-        } else {
-          setProjects([]);
-        }
-      })
+    fetchProjectsAction(tenantId)
+      .then((res) => (res.success && res.data ? res.data : []))
       .catch((error) => {
         console.error('Error fetching projects:', error);
+        return [] as Project[];
+      })
+      .then((projects) => {
         setIsLoading(false);
-        setProjects([]);
+        setProjects(projects);
       });
   }, [tenantId]);
 
+  if (isLoading) {
+    return <Skeleton className="h-12" />;
+  }
+  if (!projects.length) {
+    return <p className="px-2 text-sm text-muted-foreground">No projects yet</p>;
+  }
+
+  const project = projects.find((p) => p.projectId === projectId) as Project;
   return (
-    <div className="flex flex-col gap-2">
-      {isLoading ? (
-        <Skeleton className="h-14 w-full" />
-      ) : projects.length === 0 ? (
-        <div className="flex flex-col gap-2 px-2">
-          <p className="text-sm text-muted-foreground">No projects yet</p>
-          <NewProjectDialog tenantId={tenantId as string} />
-        </div>
-      ) : (
-        <ProjectSelector
-          projects={projects}
-          selectedProjectId={projectId as string}
-          tenantId={tenantId as string}
-        />
-      )}
-    </div>
+    <DropdownMenu>
+      <DropdownMenuTrigger asChild>
+        <SidebarMenuButton
+          size="lg"
+          className="data-[state=open]:bg-sidebar-accent data-[state=open]:text-sidebar-accent-foreground"
+        >
+          <ProjectItem
+            name={project.name}
+            description={project.description}
```

Comment:
added `project.description` instead of `tenantId`, @sarah-inkeep should I undo?
<img width="242" height="149" alt="image" src="https://github.com/user-attachments/assets/4b02f1db-1919-4eaf-b9a8-eebf9d08e418" />